packages/fast-components-react-msft/src/typography/typography.schema.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
unrelated to this file, but there should be a copy schema task we can remove.

Reply:
Doesn't look like we were ever copying the schemas to the MSFT package from Base. Looks like we are all good on this - appears to just copy the files in src to dist.